api/jsonschema/schema.json

```diff
@@ -4288,40 +4245,6 @@
       },
       "type": "object"
     },
-    "io.argoproj.workflow.v1alpha1.ArtifactResult": {
```

Comment:
This was only used by `ArtifactResultNodeStatus` which was only used by `ArtifactGCStatus`